src/runtime/auto-prompt-manager.ts

```diff
@@ -259,6 +269,7 @@ export class AutoPromptManager {
     const potentialAction = await this.getPotentialAction_({
       article,
       frequencyCapConfig,
+      shouldRenderOnsitePreview,
```

Comment:
How much of the remaining flow do we expect to be shared between preview and regular flows moving forward? 
Given that the preview flow is much simpler than the regular flow, would it be better to create a separate flow from the begining which would help maintain individual flow cleaner?

Reply:
Created a separate flow for preview prompt.